arm-datalake-analytics/catalog/2015-10-01-preview/swagger/catalog.json

```diff
@@ -976,6 +1002,153 @@
         }
       }
     },
+    "/catalog/usql/databases/{databaseName}/schemas/{schemaName}/tables/{tableName}/partitions/{partitionName}": {
+      "get": {
+        "tags": [
+          "Catalog"
+        ],
+        "operationId": "Catalog_GetTablePartition",
+        "description": "Retrieves the specified table partition from the Data Lake Analytics catalog.",
+        "parameters": [
+          {
+            "name": "databaseName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the database containing the partition."
+          },
+          {
+            "name": "schemaName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the schema containing the partition."
+          },
+          {
+            "name": "tableName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the table containing the partition."
+          },
+          {
+            "name": "partitionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the table partition."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/USqlTablePartition"
+            }
+          }
+        }
```

Comment:
@amarzavery that makes sense. Our errors actually adhere to the CloudError schema as it is right now, so it makes sense for us to use the default CloudException/CloudError setup. Below is the schema for all of our errors except Filesystem (which I updated). This should be properly deserialized by CloudError, right?

“error”: {
“code”: “string”,
“message”: “string”,
“target”: “optional string”,
“details”: [{
“code”: “string”,
“message”: “string”,
“target”: “optional string”
]},
                “innerError”: {
                                “context”: “string”,
                                “trace”: “sdf”
                                }
                }